Makefile.opensuse

```diff
@@ -5,10 +5,10 @@ AD_SERVER_SRC_FILE:=$(AD_SERVER_DIR)/Containerfile.opensuse
 CLIENT_SRC_FILE:=$(CLIENT_DIR)/Containerfile.opensuse
 TOOLBOX_SRC_FILE:=$(TOOLBOX_DIR)/Containerfile.opensuse
 SERVER_SOURCES:=$(SERVER_DIR)/smb.conf
-SERVER_NAME:=samba-opensuse-container:$(TAG)
-AD_SERVER_NAME:=samba-ad-opensuse-container:$(TAG)
-CLIENT_NAME:=samba-client-opensuse-container:$(TAG)
-TOOLBOX_NAME:=samba-toolbox-opensuse-container:$(TAG)
+SERVER_NAME:=samba-server:$(TAG)
+AD_SERVER_NAME:=samba-ad-server:$(TAG)
+CLIENT_NAME:=samba-client:$(TAG)
+TOOLBOX_NAME:=samba-toolbox:$(TAG)
```

Comment:
This will now overwrite the fedora images (notice you removed '-opensuse' from each of the names). Was this intentional?

Reply:
Yes, the "unqualified" or "local names" were inconsistent with the fully qualified (or repo) names. @obnoxxx is working on changes to make building on various bases more organized and consistent and one of the stumbling blocks we had was variable reuse was hard due to these naming differences.

We made this change together as a building block for that. So yes, right now if you run `make build-server` and following run `make -f Makefile.opensuse build-server` the short name will be overwritten. I think that OK and may even improve our ability to reuse the tests for various base images.

Reply:
Ok, you don't need these entries at all in Makefile.opensuse then. Now all we're doing is overridding the existing names with the same names.